source/clear-linux/how-to/swupd/run.rst

```diff
@@ -1,59 +1,60 @@
 .. _swupd-run:
 
-How to run the updater
-######################
+How to update Clear Linux
+#########################
 
-This section describes how to manually trigger the software update code in
-Clear Linux\* OS for Intel® Architecture.
+This section describes how to update |CLOSIA|.
 
 Prerequisites
 =============
 
--  System is up and running a Clear Linux OS for Intel
-   Architecture official build, 7420 later.
+* The system is up and running the Clear Linux 15400 build or later.
 
--  Device is on a well-connected network.
+* Device is on a well-connected network.
 
--  Device is able to connect to release infrastructure. For example:
-   http://update.clearlinux.org/update/300/
+* Device is able to connect to the release infrastructure. For example:
```

Comment:
Isn't this almost a duplicate of the previous bullet? Maybe the previous one could be "Device is on a reliable network"? Or maybe just drop the previous point and say, "Device is able to connect to http://update.clearlinux.org/" (the `update/300/` seems superfluous in this context - but if it needs to be retained, maybe change `300` to `15400` as that's mentioned above whilst `300` isn't).